echion/threads.h

```diff
@@ -26,6 +27,80 @@
 #include <echion/tasks.h>
 #include <echion/timing.h>
 
+#if defined PL_LINUX
+class ProcFdMap
```

Comment:
Can't we add an `int fd` field to `ThreadInfo` and retrieve it from there? We already have a map of these objects to track Python threads.

Reply:
Oh, I had a misconception that ThreadInfo was created for every sample() operation. Yeah that will work.

Reply:
#81 seems to work